### ISSUE TITLE
How i disable previous day selection in calender?

### ISSUE THREAD

**Issue by sanjeet007** [Mulham-Raee/Horizontal-Calendar#139]
### Expected Behavior / Goal
> ?

### Actual Behavior
> ?

### Steps to Reproduce the Problem (sample code if possible)

  >1.
  >2.
  >3.

### Specifications

 > - Android Version:
 > - Horizontal-Calendar Version:

**Comment by SumitTrivedi073** [comment]
Replace this into your calendar activity

                Calendar startDate = Calendar.getInstance();
		startDate.add(Calendar.MONTH, 0);
		Calendar endDate = Calendar.getInstance();
		endDate.add(Calendar.MONTH, 2);

**Comment by sanjeet007** [comment]
> Replace this into your calendar activity
> 
> ```
>             Calendar startDate = Calendar.getInstance();
> 	startDate.add(Calendar.MONTH, 0);
> 	Calendar endDate = Calendar.getInstance();
> 	endDate.add(Calendar.MONTH, 2);
> ```

Will it disable previous day date selection without affecting calendar

**Comment by sanjeet007** [comment]
Thanks for replying I will use this code

**Comment by sanjeet007** [comment]
One more thing can i change its ui means can i customized it's layout of horizontal calendar view?

**Comment by SumitTrivedi073** [comment]
Yes you can go to Horizontal Layout Manager and replace below mention line:- 

HorizontalLayoutManager(Context context, boolean reverseLayout) {
        super(context, HORIZONTAL, reverseLayout);
    }

**Comment by SumitTrivedi073** [comment]
yes you can but depend what you want to achieve

**Comment by darias08** [comment]
Anyway how I can disable weekends and keep it between Monday through Friday?
